WoWsShipBuilder.Common/Features/ShipStats/ShipStats.razor

```diff
@@ -246,10 +247,9 @@
         shipContainers.AddRange(buildContainers ?? indexList.Select(index => ShipBuildContainer.CreateNew(AppData.ShipDictionary[index], null, null)));
     }
 
-    private List<string> StoreBuildsForTransfer(bool calculateShipDataContainer)
+    private List<ShipBuildContainer> GetShipBuildContainers(bool calculateShipDataContainer)
```

Comment:
Consider returning an ImmutableList instead. The method's use case is not different before so this would be a good opportunity to change this.